client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfigurations.java

```diff
@@ -48,6 +48,8 @@
     private static final int DEFAULT_MAX_NUM_RETRIES = 4;
     private static final ClientConfiguration.ClientQoS DEFAULT_DISABLE_SYMPATHETIC_QOS =
             ClientConfiguration.ClientQoS.ENABLED;
+    private static final ClientConfiguration.AutomaticRetryOnQoS AUTOMATIC_RETRY_ON_QOS_DEFAULT =
```

Comment:
I named this based on your suggestion [here](https://github.com/palantir/conjure-java-runtime/pull/1041#discussion_r273715818). I'm happy to change it if you have any other ideas.